.gitignore

```diff
@@ -29,3 +29,5 @@ __pycache__
 *.pb
 coverage
 .nyc_output
+package-lock.json
```

Comment:
Is there a specific reason to ignore package-lock

Reply:
Because we have migrated to yarn, the package-lock should not be there anymore.